package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "debug": "4.3.4",
+    "discord-webhook-node": "^1.1.8",
```

Comment:
should be a fixed version since it's a third-party dependency "discord-webhook-node": "1.1.8" and rerun `npm i` to updt lockfile.